Package.swift

```diff
@@ -124,6 +124,7 @@ extension Array where Element == PackageDescription.SwiftSetting {
     availabilityMacroSettings + [
       .unsafeFlags(["-require-explicit-sendable"]),
       .enableUpcomingFeature("ExistentialAny"),
+      .enableExperimentalFeature("SuppressedAssociatedTypes"),
```

Comment:
We'll need to match this in the CMake rules too

Reply:
I thought I did. Oops.